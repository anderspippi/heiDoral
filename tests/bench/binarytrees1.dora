//= file bench/binarytrees/binarytrees.dora
//= args 9
//= stdout file