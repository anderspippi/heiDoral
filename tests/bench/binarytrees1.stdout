stretch tree of depth 10	 check: 2047
512	 trees of depth 4	 check: 15872
128	 trees of depth 6	 check: 16256
32	 trees of depth 8	 check: 16352
long lived tree of depth 9	 check: 1023
